Runtime/Action/AnyAction.cs

```diff
@@ -63,7 +63,10 @@ protected virtual void RemoveActionsListeners()
 
             foreach (Action action in Actions.SubscribableElements)
             {
-                action.ActivationStateChanged.RemoveListener(OnActionActivationStateChanged);
+                if (action)
```

Comment:
Code convention: We want to be explicit about what defines the "boolness" of objects, especially since [Unity overrides the equality of `UnityEngine.Object`s](https://blogs.unity3d.com/2014/05/16/custom-operator-should-we-keep-it/).
```suggestion
                if (action != null)
```